docker/flux-validator/README.md

```diff
@@ -41,3 +41,83 @@ Validation failed at directives:
 --noodles=2: 2
 Sep 09 06:48:51.615419 UTC 2025 broker.err[0]: rc2.0: python3 /code/docker/flux-validator/validate.py validate /data/docker/flux-validator/batch-invalid.sh Exited (rc=1) 0.1s
 ```
+
+#### Canonical jobspecs in YAML or JSON format
+
+##### Valid
+```bash
+$ docker run -it -v $(pwd):/data ghcr.io/compspec/fractale:flux-validator /data/docker/flux-validator/implicit-slot.yaml
+$ echo $?
+```
+
+##### Invalid
+```bash
+$ docker run -it -v $(pwd):/data ghcr.io/compspec/fractale:flux-validator /data/docker/flux-validator/implicit-slot-invalid.yaml
```

Comment:
Let's add `implicit-slot-invalid.yaml` to the repository too. Feel free to create additional structure for these data files if you think it will better organize.